### PR TITLE
Refactor Menu Screen

### PR DESCRIPTION
**Pull request by Sailor-Saturn**
Removed ingredients in menu screen so that the first screen only has the menu sandwiches.

### Reviewers:
@dougritter

### CODE REVIEW

File: Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift [hunk 1]
```diff
@@ -9,7 +9,6 @@ import Foundation
 
 public enum Sections: CaseIterable {
     case menuSandwiches
-    case ingredients
 }
```
Comment (carlosefonseca) [inline Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift]: Are you considering adding more sections in the future? If not, you could also remove all the Section code.

File: Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift [hunk 2]
```diff
@@ -26,40 +25,16 @@ public class TableDemoPresenter {
...
-            return menuSandwiches.count
-        case .ingredients:
-            return ingredients.count
-        }
-    }
-    
-    func getSectionHeaders() -> [Sections]{
-        return Sections.allCases
+    func numberOfRowsInSection() -> Int {
+        return menuSandwiches.count
```
Comment (carlosefonseca) [inline Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift:34]: `TableDemoViewPresenterTests` is quite tiny, no tests validate the expected rows and sections of the menu 😛

File: Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift (hunk 1 above)
Comment (Sailor-Saturn) [reply Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift]: Yes I agree, I'll do it now.

**Review by carlosefonseca — APPROVED** [review]
👍 Check the comments but I think it's good to go

File: Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift (hunk 2 above)
Comment (Sailor-Saturn) [reply Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift:34]: Hey Carlos, that's true, can I do the tests in other occasion?

File: Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift [hunk 3]
```diff
@@ -26,7 +23,7 @@ public class TableDemoPresenter {
     }()
     
     func numberOfSections() -> Int {
-        return Sections.allCases.count
+        return 1
     }
```
Comment (carlosefonseca) [inline Lanchonete/Lanchonete/ Features/Menu/Presentation/TableDemoViewPresenter.swift:27]: You may remove the method if you want, returning 1 is the default behavior 🙂

**Pull request merged**